vars/utils.groovy

```diff
@@ -308,26 +321,60 @@ def publishCondaEnv(jobconfig, test_info) {
 }
 
 
+// Validates an XML file against xunit schema
```

Comment:
Okay I have removed this. I'll open another pr to add that utility function and it can be decided if its needed. Should I squash my commits to clean up this PR?